regolith/chained_db.py

```diff
@@ -15,7 +15,7 @@ def __getitem__(self, key):
         results = []
         # Try to get all the data from all the mappings
         for mapping in self.maps:
-            results.append(mapping.get(key, None))
+            results.append(mapping.get(key, "--bad-request--"))
```

Comment:
I think you should define a different sentinal, something that can't possibly be a valid value,and preferably as singleton.

Check out xonsh's DefaultNotGiven for an example: https://github.com/xonsh/xonsh/blob/master/xonsh/tools.py#L232